### ISSUE TITLE
Toolbox mxt fails loading on Moneydance Preview release 2021.1 (3051)

### ISSUE THREAD

**Issue by peterlynch** [yogi1967/MoneydancePythonScripts#6]
Thanks for the great tools. 

I encountered a problem launching the Toolbox mxt on Moneydance preview release 2021.1 (3051)

```
toolbox.py(Extension):2021/03/17-07:38:31: Checking encryption key....
loading with 128 bit encryption key
Traceback (most recent call last):
  File "extension(toolbox):toolbox.py", line 17433, in <module>
  File "extension(toolbox):toolbox.py", line 16959, in openDisplay
  File "extension(toolbox):toolbox.py", line 2174, in buildDiagText
  File "extension(toolbox):toolbox.py", line 4916, in get_orphaned_extension
AttributeError: 'com.moneydance.apps.md.controller.Main' object has no attribute 'getOutdatedExtensionIDs'
```

**Comment by yogi1967** [comment]
Build 3051. Released today? Hadn’t seen it. Thx. Will check it out. 

Regards Stuart B

On 17 Mar 2021, at 10:44, Peter Lynch ***@***.***> wrote:

﻿
Thanks for the great tools.

I encountered a problem launching the Toolbox mxt on Moneydance preview release 2021.1 (3051)

toolbox.py(Extension):2021/03/17-07:38:31: Checking encryption key....
loading with 128 bit encryption key
Traceback (most recent call last):
  File "extension(toolbox):toolbox.py", line 17433, in <module>
  File "extension(toolbox):toolbox.py", line 16959, in openDisplay
  File "extension(toolbox):toolbox.py", line 2174, in buildDiagText
  File "extension(toolbox):toolbox.py", line 4916, in get_orphaned_extension
AttributeError: 'com.moneydance.apps.md.controller.Main' object has no attribute 'getOutdatedExtensionIDs'

**Comment by yogi1967** [comment]
A lot broke with build 3051. I have fixed everything. Now in testing. Thanks.  

Regards Stuart

On 17 Mar 2021, at 10:44, Peter Lynch ***@***.***> wrote:

﻿
Thanks for the great tools.

I encountered a problem launching the Toolbox mxt on Moneydance preview release 2021.1 (3051)

toolbox.py(Extension):2021/03/17-07:38:31: Checking encryption key....
loading with 128 bit encryption key
Traceback (most recent call last):
  File "extension(toolbox):toolbox.py", line 17433, in <module>
  File "extension(toolbox):toolbox.py", line 16959, in openDisplay
  File "extension(toolbox):toolbox.py", line 2174, in buildDiagText
  File "extension(toolbox):toolbox.py", line 4916, in get_orphaned_extension
AttributeError: 'com.moneydance.apps.md.controller.Main' object has no attribute 'getOutdatedExtensionIDs'
